application.py

```diff
@@ -1,24 +1,31 @@
-import argparse
-import os
-
-from app import application
-
-# Only enable Flask debugging if an env var is set to true
-application.debug = os.environ.get('FLASK_DEBUG') in ['true', 'True']
-
-# Get application version from env
-app_version = os.environ.get('APP_VERSION')
-
-# Get cool new feature flag from env
-enable_cool_new_feature = os.environ.get('ENABLE_COOL_NEW_FEATURE') in ['true', 'True']
-
+"""
+Application Entry
+====================
+"""
 
 if __name__ == '__main__':
+    import argparse
+
     parser = argparse.ArgumentParser(description='arguments options:')
     parser.add_argument(
         '-p', '--port', type=int, default=5000, help='Specify listening port.'
     )
     args = parser.parse_args()
     port = args.port
 
-    application.run(host='0.0.0.0', port=port)
+    import logging.config
+    import yaml
+    import os
+
+    if not os.path.exists('./logs'):
+        os.makedirs('./logs')
+
+    with open('logging-conf.yaml', 'r') as f:
+        config = yaml.safe_load(f.read())
+        logging.config.dictConfig(config)
+
+    from app import create_app
+
+    app = create_app()
+
+    app.run(host='127.0.0.1', port=port)
```

Comment:
I referred to [this skeleton](https://github.com/aws-samples/eb-python-flask/blob/master/application.py) here, so I'm not sure if this change breaks. I do not entirely understand the diff between 0.0.0.0 and 127.0.0.1  Do you?

Reply:
0.0.0.0 is different in windows.

Reply:
Seems 0.0.0.0 is a must. https://stackoverflow.com/a/52185250